### PR TITLE
add documention how to rotate access tokens

### PR DESCRIPTION
**Pull request by qeqar**

### CODE REVIEW

File: docs/_gks/accessmanagement/index.de.md [hunk 1]
```diff
@@ -33,6 +33,18 @@ Benutzer mit diesem Zugriffslevel wiederum haben keinen Zugriff auf das GKS Dash
 
 Mehr Informationen über Kubernetes RBAC finden Sie [hier](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
 
+## Login Tokens resetten
+
+Zum rotieren der Login token.
```
Comment (kllex) [inline docs/_gks/accessmanagement/index.de.md:38]:
```suggestion
Zum Rotieren des Login-Tokens.
```

File: docs/_gks/accessmanagement/index.de.md [hunk 2]
```diff
@@ -33,6 +33,18 @@ Benutzer mit diesem Zugriffslevel wiederum haben keinen Zugriff auf das GKS Dash
...
+
+![Revoke Menu](revoke-token-menu.png)
+
+Hier den Token auswählen und den revoke button klicken.
+
+![Revoke Popup](revoke.png)
+
+Anschließend eine neue [Kubeconfig][#Zugriff via GKS-Projekt] herunterladen, die alte ist jetzt ungültig.
```
Comment (kllex) [inline docs/_gks/accessmanagement/index.de.md:46]:
```suggestion
Anschließend eine neue [Kubeconfig][#Zugriff via GKS-Projekt] herunterladen, da die Vorherige jetzt ungültig ist.
```

File: docs/_gks/accessmanagement/index.de.md [hunk 3]
```diff
@@ -33,6 +33,18 @@ Benutzer mit diesem Zugriffslevel wiederum haben keinen Zugriff auf das GKS Dash
 
 Mehr Informationen über Kubernetes RBAC finden Sie [hier](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
 
+## Login Tokens resetten
+
+Zum rotieren der Login token.
+
+![Revoke Menu](revoke-token-menu.png)
+
+Hier den Token auswählen und den revoke button klicken.
```
Comment (kllex) [inline docs/_gks/accessmanagement/index.de.md:42]:
```suggestion
Nun den Token auswählen und den revoke Knopf drücken.
```

File: docs/_gks/accessmanagement/index.en.md [hunk 4]
```diff
@@ -32,6 +32,18 @@ A user with this level of access can download a specific `kubeconfig`, which can
 
 More information on Kubernetes RBAC is available [here](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
 
+## Revoke Tokens
+
+If you need to rotate the Kubeconfig Login tokens.
+
+![Revoke Menu](revoke-token-menu.png)
+
+Then select the Token and press revoke
```
Comment (kllex) [inline docs/_gks/accessmanagement/index.en.md:41]:
```suggestion
Then select the token and press revoke.
```

File: docs/_gks/accessmanagement/index.en.md [hunk 5]
```diff
@@ -32,6 +32,18 @@ A user with this level of access can download a specific `kubeconfig`, which can
...
+
+![Revoke Popup](revoke.png)
+
+Then download new [Kubeconfig][#Project-Based Access], the old one is now invalid.
```
Comment (kllex) [inline docs/_gks/accessmanagement/index.en.md:45]:
```suggestion
Then download the new [Kubeconfig][#Project-Based Access] as the old one is now invalid.
```

**Comment by kllex** [comment]
Both screenshots contain the name of the project "Mark's Test Project", which is not in line with the rest of our screenshot's.

**Comment by qeqar** [comment]
> Both screenshots contain the name of the project "Mark's Test Project", which is not in line with the rest of our screenshot's.

done

**Review by syjabri — APPROVED** [review]

**Review by kllex — APPROVED** [review]

**Pull request merged**